cmd/login/flag.go

```diff
@@ -52,6 +56,8 @@ func (f *flag) Init(cmd *cobra.Command) {
 	cmd.Flags().StringVar(&f.WCCertCNPrefix, flagWCCertCNPrefix, "", fmt.Sprintf(`For client certificate creation. Prefix for the name encoded in the X.509 field "CN". Requires --%s.`, flagWCName))
 	cmd.Flags().BoolVar(&f.WCInsecureNamespace, flagWCInsecureNamespace, false, fmt.Sprintf(`For client certificate creation. Allow using an insecure namespace for creating the client certificate. Requires --%s.`, flagWCName))
 
+	cmd.Flags().StringVar(&f.ConnectorID, flagConnectorID, "", "Identifier of a specific Dex connector to be used for authentication. The connector is selected during the oidc authentication process by default. If this flag is specified, the selection step is skipped.")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&f.ConnectorID, flagConnectorID, "", "ID of the Dex connector to use for authentication. With more than one connector configured in Dex, this allows to skip the selection page.")
```

Reply:
It would skip either way, though, wouldn't it? Even if only one connector is there.

Reply:
Yep, right. Simplified it more.